i18n/pl.json

```diff
@@ -178,19 +178,20 @@
 			["content_team_member", "Członek Zespołu Treści"],
 			["GRASP", "GRASP"],
 			["bot-global", "bot Fandom"],
-			["staff", "personel Fandom"],
+			["staff", "personel Fandomu"],
 			["wiki-manager", "Menedżer wiki"],
-			["helper", "pomocnik Fandom"],
+			["helper", "pomocnik Fandomu"],
```

Comment:
Fandom Staff and Helpers groups aren't localized and neither they should be. Please leave it as "Fandom Staff" for Staff members and "Helper" for Helpers.